sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -116,9 +116,9 @@ char *ottf_task_get_self_name(void);
     uint64_t t_start_ = ibex_mcycle_read();                              \
     status_t local_status = INTO_STATUS(test_function_());               \
     uint64_t t_end_ = ibex_mcycle_read();                                \
-    CHECK(t_end_ <= UINT32_MAX);                                         \
-    CHECK(t_start_ <= t_end_);                                           \
-    uint32_t cycles_ = (uint32_t)t_end_ - (uint32_t)t_start_;            \
+    uint64_t cycles64_ = t_end_ - t_start_;                              \
+    CHECK(cycles64_ <= UINT32_MAX);                                      \
```

Comment:
```suggestion
    CHECK(cycles_ <= UINT32_MAX);                                      \
```

Reply:
Thanks, updated with the shorter macro

FYI GitHub lets you select multiple line numbers, then you can do a suggestion over all of them in one comment